### ISSUE TITLE
Redirect Request: Family member hub phase 4 (Survivor compensation)

### ISSUE THREAD

**Issue by kristinoletmuskat** [department-of-veterans-affairs/va.gov-team#91851]
| POC | Github alias(es) |
--- | --- 
| Content point of contact: | @d-beyer  |
| IA point of contact: | @kristinoletmuskat  |

### Type of request
- [x] We are retiring or taking down a page and need to redirect the URL (complete redirect section)
- [x] We are changing the URL of an existing page (complete redirect section)
- [ ] We need a custom vanity URL (complete vanity URL section)
- [ ] We are removing a temporary redirect  (add relevant background and link to original redirect request)

### Do you have a product technical team able to implement this redirect?
- [ ] Yes - Please note: Redirect still needs approval from IA.
  - [ ] Please verify your team has reviewed [Redirect governance](https://design.va.gov/components/url-standards/redirects) and [Redirect technical implementation information
](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/engineering/redirect-implementation-strategy.md) 
- [x] No - CAIA will need to make Public Websites aware of the ticket.

### Implementation date
When does this request need to be live? **NOTE:** Redirects in vsp-platform-revproxy auto-deploy M-Th. Friday launches should be avoided.
Choose one:
- [x] On a specific date, connected to a program or publishing requirement: Still TBD, but aiming for maybe late September
- [ ] Any day prior to a deadline/latest possible date. **DEADLINE:** 
- [ ] Any day after an earliest possible date. **EARLIEST DATE:**
- [ ] During a specific date range: ***EARLIEST DATE TO LATEST DATE:***
- [ ] No hard timing requirements, publish when possible.

### Redirects 
Current URL  |  Redirect Destination or New URL
| Page                                        | Current URL                                                         | Redirect to                                                                                                                 | Status                                                         |   |
|---------------------------------------------|---------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------|---|
| DIC                                         | https://www.va.gov/disability/dependency-indemnity-compensation/    | va.gov/family-and-caregiver-benefits/survivor-compensation/dependency-indemnity-compensation/                               | H1 TBD by content                                              |   |
| DIC - Survivor rates                        | https://www.va.gov/disability/survivor-dic-rates/                   | va.gov/family-and-caregiver-benefits/survivor-compensation/dependency-indemnity-compensation/survivor-rates/                | H1 TBD by content                                              |   |
| DIC - Survivor rates - past 2023            | https://www.va.gov/disability/survivor-dic-rates/past-rates-2023/   | va.gov/family-and-caregiver-benefits/survivor-compensation/dependency-indemnity-compensation/survivor-rates/past-rates-2023 | H1 TBD by content                                              |   |
| DIC - Survivor rates - past 2022            | https://www.va.gov/disability/survivor-dic-rates/past-rates-2022/   | va.gov/family-and-caregiver-benefits/survivor-compensation/dependency-indemnity-compensation/survivor-rates/past-rates-2022 | H1 TBD by content                                              |   |
| DIC - Survivor rates - past 2021            | https://www.va.gov/disability/survivor-dic-rates/past-rates-2021/   | va.gov/family-and-caregiver-benefits/survivor-compensation/dependency-indemnity-compensation/survivor-rates/past-rates-2021 | H1 TBD by content                                              |   |
| DIC - Parent rates                          | https://www.va.gov/disability/parent-dic-rates/                     | va.gov/family-and-caregiver-benefits/survivor-compensation/dependency-indemnity-compensation/parent-rates/                  | H1 TBD by content                                              |   |
| DIC - Parent rates - past 2023              | https://www.va.gov/disability/parent-dic-rates/past-rates-2023/     | va.gov/family-and-caregiver-benefits/survivor-compensation/dependency-indemnity-compensation/parent-rates/past-rates-2023   | H1 TBD by content                                              |   |
| DIC - Parent rates - past 2022              | https://www.va.gov/disability/parent-dic-rates/past-rates-2022/     | va.gov/family-and-caregiver-benefits/survivor-compensation/dependency-indemnity-compensation/parent-rates/past-rates-2022   | H1 TBD by content                                              |   |
| DIC - Parent rates - past 2021              | https://www.va.gov/disability/parent-dic-rates/past-rates-2021/     | va.gov/family-and-caregiver-benefits/survivor-compensation/dependency-indemnity-compensation/parent-rates/past-rates-2021   | H1 TBD by content                                              |   |
|                           Survivors Pension | https://www.va.gov/pension/survivors-pension/                       | va.gov/family-and-caregiver-benefits/survivor-compensation/survivors-pension/                                               | H1 TBD by content                                              |   |
|                   Survivors Pension - rates | https://www.va.gov/pension/survivors-pension-rates/                 | va.gov/family-and-caregiver-benefits/survivor-compensation/survivors-pension/rates/                                         | H1 TBD by content                                              |   |
| Survivors Pension - rates - past rates 2023 | https://www.va.gov/pension/survivors-pension-rates/past-rates-2023/ | va.gov/family-and-caregiver-benefits/survivor-compensation/survivors-pension/rates/past-rates-2023                          | H1 TBD by content                                              |   |
| Survivors Pension - rates - past rates 2022 | https://www.va.gov/pension/survivors-pension-rates/past-rates-2022/ | va.gov/family-and-caregiver-benefits/survivor-compensation/survivors-pension/rates/past-rates-2022                          | H1 TBD by content                                              |   |
| Survivors Pension - rates - past rates 2021 | https://www.va.gov/pension/survivors-pension-rates/past-rates-2021/ | va.gov/family-and-caregiver-benefits/survivor-compensation/survivors-pension/rates/past-rates-2021                          | H1 TBD by content                                              |   |

**Link to campaign landing page request issue, if relevant:**

### Process, Roles and Responsibilities
- [x] IA: Above information is provided
- [ ] IA: All appropriate VA stakeholders are notified as appropriate
- [x] IA: Request is vetted and documented and implementation plan is clear 
- [x] IA: Request is assigned to appropriate team for implementation 
- [ ] Implementation team: Work is complete
- [ ] Implementation team: Validated in production
- [ ] Requesting team: Validates everything is correct in production and closes ticket
- [ ] IA: create content ticket to correct nodes in Drupal

**Comment by FranECross** [comment]
@kristinoletmuskat Acknowledging assignment and that we're awaiting info on the release date.

**Comment by jilladams** [comment]
@kristinoletmuskat we plan sprint for end of September tomorrow (Wed 9/18). Can you confirm date on this one?

**Comment by kristinoletmuskat** [comment]
Hey @jilladams sorry I missed this one! I'm guessing mid October at the latest.

**Comment by kristinoletmuskat** [comment]
Hey @FranECross @jilladams  an update on this one:

1 we are going to split this work into 2 phases due to our stakeholder review timeline. So there will be a 4A and 4B ticket. They don't have any dependencies on eachother, just a heads up. I'll make those changes to the tickets soon. 
2. We'd like to propose the go-live for Tuesday, 10/8. Would that work for you all?

**Comment by FranECross** [comment]
@kristinoletmuskat Thanks for the update! Yes, 10/8 go live works great.

**Comment by FranECross** [comment]
@kristinoletmuskat Just checking to be sure this is still a go for 10/8 for all the redirects shown in the ticket?  I till see under status "H1 TBD by content", and this checkbox isn't checked "IA: All appropriate VA stakeholders are notified as appropriate". Thanks!

**Comment by kristinoletmuskat** [comment]
Hey @FranECross yes we are still targeting next week, and are finalizing these last pieces. I'm following up with content about our stakeholders and the H1s today.

**Comment by kristinoletmuskat** [comment]
hiiiii @FranECross stakeholders have been notified and all URLs finalized. We are ready to go for Tuesday.

**Comment by FranECross** [comment]
Thanks, @kristinoletmuskat !!!

**Comment by kristinoletmuskat** [comment]
Hey @FranECross, we had to delay our publishing step because we have a copyeditor out due to the hurricane :( 

Would it be possible to merge the redirects on Wednesday (10/9) to deploy on Thursday (10/10) instead? We think we can publish by Tuesday (10/8) EOD.

**Comment by jilladams** [comment]
Fran is out today, but yes, that's fine (FYI @chriskim2311 ) and I've updated date in ticket title / body.

**Comment by kristinoletmuskat** [comment]
Hey @FranECross , just a note that I added one URL at the end for the spanish version of the Survivor Pension page. 

Once we redirect, the URL should work for this page, but there WONT be a toggle or left nav for this yet. The page is still going through the QA process for translated pages with the state department, and once that is done, we'll submit a separate ticket for the toggle and left nav. I'm not sure if you need to know this info! But just wanted to explain why this page isn't included in our toggle/left nav ticket.

**Comment by FranECross** [comment]
@kristinoletmuskat Thanks so much for the head's up!

**Comment by kristinoletmuskat** [comment]
Hey @FranECross I just wanted to check whether y'all were able to merge these today?

**Comment by jilladams** [comment]
The work is in PR but not yet merged: https://github.com/department-of-veterans-affairs/vsp-platform-revproxy/pull/677

Should merge today for tomorrow deploy.

**Comment by chriskim2311** [comment]
Merged and will go out with tomorrow's release

**Comment by kristinoletmuskat** [comment]
Hello! Confirmed that all of the redirects are working except for:


Survivors Pension - rates - past rates 2021 | https://www.va.gov/pension/survivors-pension-rates/past-rates-2021/ | va.gov/family-and-caregiver-benefits/survivor-compensation/survivors-pension/rates/past-rates-2021
-- | -- | --

It looks like that was accidentally saved in draft, so I just published and it should work.
